inspire_dojson/utils/__init__.py

```diff
@@ -98,6 +98,19 @@ def absolute_url(relative_url):
     return urllib.parse.urljoin(server, relative_url)
 
 
+def afs_url(eos_path):
+    """Returns the afs absolute path from eos path.
+
+    The base URL is taken from the Flask app config if present, otherwise it
+    falls back to ``http://inspirehep.net``.
+    """
+    default_afs_path = '/afs/cern.ch/project/inspire/PROD/'
+    file_path = current_app.config.get('FILE_SERVER_PATH', default_afs_path)
```

Comment:
Why do we need this configurability? We already know in advance which path we're going to get.

We should remove, not reinforce, the dependency of `inspire-dojson` from Flask.